template.nunjucks

```diff
@@ -17,23 +17,23 @@
       data-site-assetpath="{{ site.assetPath }}"
       data-components="{{ _components }}">
       <div class="kiln-toolbar-inner">
-        <button class="user-icon">{% include 'public/media/components/clay-kiln/user-icon.svg' %}</button>
+        <button class="user-icon">{% include 'public/media/components/clay-kiln/user-icon.svg' ignore missing %}</button>
```

Comment:
why `ignore missing`? If these icons are missing, we've broken something horribly.